Botkit/Microsoft.BotKit/BotWorker.cs

```diff
@@ -1,96 +1,107 @@
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Dynamic;
+    using System.Threading.Tasks;
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Builder.Dialogs;
+    using Microsoft.Bot.Connector;
+    using Microsoft.Bot.Schema;
+    using Microsoft.BotKit.Core;
+
     /// <summary>
     /// A base class for a `bot` instance, an object that contains the information and functionality for taking action in response to an incoming message.
     /// Note that adapters are likely to extend this class with additional platform-specific methods - refer to the adapter documentation for these extensions.
     /// </summary>
     public class BotWorker
     {
-        public BotWorkerConfiguration config;
-
-        public Botkit Controller { get; }
+        private BotWorkerConfiguration config;
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="BotWorker"/> class.
         /// Create a new BotWorker instance. Do not call this directly - instead, use controller.spawn().
         /// </summary>
-        /// <param name="controller">A pointer to the main Botkit controller</param>
-        /// <param name="config">An object typically containing { dialogContext, reference, context, activity }</param>
+        /// <param name="controller">A pointer to the main Botkit controller.</param>
+        /// <param name="config">An object typically containing { dialogContext, reference, context, activity }.</param>
         public BotWorker(Botkit controller, BotWorkerConfiguration config)
         {
-            Controller = controller;
+            this.Controller = controller;
             this.config = config;
         }
 
+        /// <summary>
+        /// Gets Controller of BotWorker.
+        /// </summary>
+        public Botkit Controller { get; }
+
+        /// <summary>
         /// Send a message using whatever context the bot was spawned in or set using changeContext() --
-        /// or more likely, one of the platform-specific helpers like startPrivateConversation() (Slack), startConversationWithUser() (Twilio SMS), and 
+        /// or more likely, one of the platform-specific helpers like startPrivateConversation() (Slack), startConversationWithUser() (Twilio SMS), and
         /// startConversationWithUser() (Facebook Messenger) Be sure to check the platform documentation for others - most adapters include at least one.
         /// </summary>
-        /// <param name="message">A BotkitSlackMessage containing the text of a reply, or more fully formed message object</param>
-        /// <returns>Return value will contain the results of the send action, typically {id: <id of message>}</returns>
+        /// <param name="message">A BotkitSlackMessage containing the text of a reply, or more fully formed message object.</param>
+        /// <returns>A <see cref="Task{TResult}"/>Return value will contain the results of the send action, typically {id:. <id of message>}.</returns>
```

Comment:
Is there an extra '.' here?

Reply:
Yes, there is an extra '.' there. But if I remove it, then a warning is shown saying: Documentation text should end with a period. Do you think we can ignore that warning?

Reply:
No, just leave it then. No problem.